### PR TITLE
[DO NOT MERGE] - Update the Keeping Ballerina Up To Date Guide for Swan Lake

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the "Keeping Ballerina Up To Date" guide[1] for the Swan Lake release.

[1] https://ballerina.io/swan-lake/learn/keeping-ballerina-up-to-date
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 1]
```diff
@@ -56,26 +55,13 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
 
 ### Release channels
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
-
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
+Ballerina distributions of the Swan Lake channel are currently released as preview releases.
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:58]: Still we have 2 channels (Swan Lake and 1.x channel)
Still we will be doing patch releases to 1.x channel

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 2]
```diff
@@ -147,30 +133,16 @@ Use 'ballerina help dist <command>' for more information on a specific command.
 
 Most of these subcommands are self-explanatory. Therefore, the following sections introduce them briefly.
 
-### Update to the latest patch version
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:150]: This section is still valid as we will be doing 1.2.x releases and it will still update to latest patch.
In swan lake it will bump to next preview

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 3]
```diff
@@ -231,27 +222,15 @@ Distributions available locally:
...
 
 ```sh
-→ sudo ballerina dist pull 1.0.3
-Fetching the '1.0.3' distribution from the remote server...
-Downloading 1.0.3 100% [==================================] 96/96 MB '1.0.3' successfully set as the active distribution
+→ ballerina dist remove 1.0.5
+Distribution '1.0.5' successfully removed
 ```
 
-### Update the Ballerina tool
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:244]: Why this section is removed?

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 4]
```diff
@@ -56,26 +55,13 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
...
-This channel gives you access to feature releases of Ballerina distributions. Ballerina programs that you’ve written today should continue to work on these minor releases. There will be four minor releases a year. You will get access to a minor release on the 3rd Wednesday of the 3rd month of every quarter.
-
-*Example minor releases: 1.1.0, 1.2.0, 1.3.0*
+For example: Swan Lake Preview 1
 
 #### Release maintenance
 
-- We maintain a minor release 1.x.0 by issuing a series of patch releases 1.x.y. The maintenance of a particular minor release stops when there are two newer minor releases available.
-- In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, maintaining 1.0.0 will be stopped.
+The maintenance of a particular preview release stops when the stable version of it is available. For example, the preview releases of Swan Lake will stop when the stable Swan Lake GA version is released.
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:64]: In preview releases we don't have anything for maintenance. Once GA released its a stable release. Until 1.2.x will be maintained

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 5]
```diff
@@ -56,26 +55,13 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
...
 ### Release channels
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
-
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
+Ballerina distributions of the Swan Lake channel are currently released as preview releases. 
 
-#### Patch release channel
-
-This channel gives you access to the patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
```
Comment (keizer619) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:65]: This is still there for 1.2.x releases

**Comment by praneesha** [comment]
Closing as these updates are no longer required.
